src/hpr_routing.erl

```diff
@@ -7,9 +7,23 @@
     handle_packet/1
 ]).
 
+-ifdef(TEST).
+
+-export([
+    find_routes/2
```

Comment:
Can we export this normally? 
I was having to reconstruct this function in the REPL the other day while I was investigating some routing.

Reply:
We can